### DISCUSSION TITLE
Add a function to sense if someone is right clicking, left clicking or using the middle mouse

### DISCUSSION THREAD

**Discussion by Katamarang** [ScratchAddons/ScratchAddons#3853] [Ideas]
Just like the title suggests, it would be a peice of code in the sensing section that would sense if the right click, left click or middle mouse is clicked. It would be a mix between the mouse down block and the if key pressed block

**Comment by Samq64** [comment]
Scratch Addons is a browser extension and everything it does to the `.sb3` file has to work for non SA users. TurbWarp implemented this block a while back, but it won't be adding any more new blocks in the future and it's mostly just there for backward compatibility, but you can use it if you want.
